# Transforms for the laser scanners from the robot's base_link frame

laser_scanner1:
    pose:
        x: 0.0
        y: 0.0
        z: 0.0
        qx: 0.0
        qy: 0.0
        qz: 0.0
        qw: 1.0
    frames:
        frame_id: "base_link"
        child_frame_id: "laser_scanner1_link"
